app/components/funding_widget_component.html.erb

```diff
@@ -5,26 +5,39 @@
     <div class="call-to-action__text">
       <div class="call-to-action__form">
         <%= form_with(model: form, url: path, method: :get, local: true, html: { data: { "controller": "funding-widget", "funding-widget-target": 'form'}}) do |f| %>
+          <% if form.errors.any? %>
+            <div class="govuk-error-summary" data-module="govuk-error-summary">
+              <div role="alert">
+                <h2 class="govuk-error-summary__title">There is a problem</h2>
+                <div class="govuk-error-summary__body">
+                  <ul class="govuk-list govuk-error-summary__list">
```

Comment:
The error listed has white font colour when it receives focus @martyn-w 

![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/0513bc5f-e0d4-42df-924e-556d3683cdfa)